media-gfx/hydrus/hydrus-440.ebuild

```diff
@@ -0,0 +1,149 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{3..8} )
+
+inherit eutils python-single-r1 desktop
+
+DESCRIPTION="A booru-like media organizer for the desktop"
+HOMEPAGE="http://hydrusnetwork.github.io/hydrus/ https://github.com/hydrusnetwork/hydrus"
+SRC_URI="https://github.com/hydrusnetwork/${PN}/archive/v${PV}.tar.gz -> ${PF}.tar.gz"
+
+IUSE="+mpv +ffmpeg miniupnpc +lz4 socks +cloudscraper charts test"
+
+# hydrus itself is WTFPL
+# icons included are CC-BY-2.5
+LICENSE="WTFPL-2 CC-BY-2.5"
+SLOT="0"
+KEYWORDS="~amd64"
+
+RESTRICT="!test? ( test )"
+
+RDEPEND="$(python_gen_cond_dep '
+	dev-python/beautifulsoup:4[${PYTHON_MULTI_USEDEP}]
+	dev-python/chardet[${PYTHON_MULTI_USEDEP}]
+	cloudscraper? ( dev-python/cloudscraper[${PYTHON_MULTI_USEDEP}] )
+	dev-python/html5lib[${PYTHON_MULTI_USEDEP}]
+	dev-python/lxml[${PYTHON_MULTI_USEDEP}]
+	lz4? ( dev-python/lz4[${PYTHON_MULTI_USEDEP}] )
+	dev-python/numpy[${PYTHON_MULTI_USEDEP}]
+	media-libs/opencv[python,${PYTHON_MULTI_USEDEP}]
+	dev-python/pillow[${PYTHON_MULTI_USEDEP}]
+	dev-python/psutil[${PYTHON_MULTI_USEDEP}]
+	dev-python/pyopenssl[${PYTHON_MULTI_USEDEP}]
+	dev-python/pyside2[widgets,gui,charts?,${PYTHON_MULTI_USEDEP}]
+	socks? (
+			|| ( dev-python/requests[socks5,${PYTHON_MULTI_USEDEP}]
+				dev-python/PySocks[${PYTHON_MULTI_USEDEP}] )
+	)
+	mpv? (
+		 media-video/mpv[libmpv,${PYTHON_MULTI_USEDEP}]
+		 dev-python/python-mpv[${PYTHON_MULTI_USEDEP}]
+	)
+	dev-python/pyyaml[${PYTHON_MULTI_USEDEP}]
+	>=dev-python/QtPy-1.9.0-r4[pyside2,${PYTHON_MULTI_USEDEP}]
+	dev-python/requests[${PYTHON_MULTI_USEDEP}]
+	dev-python/send2trash[${PYTHON_MULTI_USEDEP}]
+	dev-python/service_identity[${PYTHON_MULTI_USEDEP}]
+	dev-python/six[${PYTHON_MULTI_USEDEP}]
+	dev-python/twisted[${PYTHON_MULTI_USEDEP}]
+
+	ffmpeg? ( media-video/ffmpeg )
+	miniupnpc? ( net-libs/miniupnpc )
+	')
+	${PYTHON_DEPS}
+"
+BDEPEND="${RDEPEND}
+	$(python_gen_cond_dep '
+	test? (
+		dev-python/mock[${PYTHON_MULTI_USEDEP}]
+		dev-python/httmock[${PYTHON_MULTI_USEDEP}]
+		dev-python/unittest2[${PYTHON_MULTI_USEDEP}]
+		dev-python/nose[${PYTHON_MULTI_USEDEP}]
+	)
+	')
+"
+
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+# Delete files only needed for testing
+delete_test_files() {
+	rm test.py
+	rm hydrus/hydrus_test.py
+	rm -r hydrus/test/
+	rm -r static/testing/
+}
+
+src_prepare() {
+	eapply "${FILESDIR}/userpath-in-local-share.patch"
```

Comment:
Move to `PATCHES=(...)`, delete `eapply` and `eapply_user`, and replace with `default`.